lesson_03/homework_03.py

```diff
@@ -0,0 +1,167 @@
+alice_in_wonderland = ('"Would you tell me, please, which way I ought to go from here?"\n'
+                       '"That depends a good deal on where you want to get to," said the Cat.\n'
+                       '"I don\'t much care where" said Alice.\n'
+'"Then it doesn\'t matter which way you go," said the Cat.\n'
+'"So long as I get somewhere," Alice added as an explanation.\n'
+'"Oh, you\'re sure to do that," said the Cat,\n'
+'"If you only walk long enough."')
+
+for single_quote in alice_in_wonderland:
+    if single_quote == "'":
+        print(single_quote)
+
+print(f"{alice_in_wonderland}")
+
+# task 01 == Розділіть змінну alice_in_wonderland так, щоб вона займала декілька фізичних лінії
+# task 02 == Знайдіть та відобразіть всі символи одинарної лапки (') у тексті
+# task 03 == Виведіть змінну alice_in_wonderland на друк
+
+
+"""
+    # Задачі 04 -10:
+    # Переведіть задачі з книги "Математика, 5 клас"
+    # на мову пітон і виведіть відповідь, так, щоб було
+    # зрозуміло дитині, що навчається в п'ятому класі
+"""
+# task 04
+"""
+Площа Чорного моря становить 436 402 км2, а площа Азовського
+моря становить 37 800 км2. Яку площу займають Чорне та Азов-
+ське моря разом?
+"""
+black_sea_area = 436402
+azov_sea_area = 37800
+total_sea_area = black_sea_area + azov_sea_area
+print(f"Загальна площа Чорного та Азовського морыв разом: {total_sea_area}")
+
+# task 05
+"""
+Мережа супермаркетів має 3 склади, де всього розміщено
+375 291 товар. На першому та другому складах перебуває
+250 449 товарів. На другому та третьому – 222 950 товарів.
+Знайдіть кількість товарів, що розміщені на кожному складі.
+"""
+total_product = 375291
+first_storage_and_second_storage = 250449
+second_storage_and_third_storage = 222950
+
+third_storage = total_product - second_storage_and_third_storage

```

Comment:
I guess your calculation is wrong: if you subtract from total_product second and third - than you leave first storage items, NOT third. In this case your calculations below can be wrong.